packages/theme/package.json

```diff
@@ -50,6 +50,7 @@
     "yaml": "2.7.0"
   },
   "devDependencies": {
+    "@shopify/theme-hot-reload": "^0.0.5",
```

Comment:
Do you think we should version the events here somehow? (in runtime)

This way, if we need to change something, we can prevent older versions of the CLI from breaking (we will likely need to keep triggering old events if we really change the shape of them)